### PR TITLE
Katie test branch

### PR DESCRIPTION
**Pull request by nc-kalmk**
Added a small file

### CODE REVIEW

File: universe.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+def hello_world(speaker=""):
```
Comment (nc-hurleyk) [inline universe.py]: Generally we like to put in some space for assignments:  
```py
def hello_world(speaker = "")
```

File: universe.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+		speaker = "{}: ".format(speaker)
+	print "{}Hello World!".format(speaker)
+
+def alien_world(person):
+	## This is a method that speaks for aliens!
+	hello_world(person)
+	print "Alien: We come in peace!"
+
+person = input("Enter your name in quotes:")
+alien_world(person)
```
Comment (nc-hurleyk) [inline universe.py]: NP:  insert new line after final line.

**Review by nc-hurleyk — CHANGES_REQUESTED** [review]
Some changes for you.  :)

**Review by nc-hurleyk — APPROVED** [review]
🥓

**Comment by nc-kalmk** [comment]
@nc-hurleyk How do I know that squashing worked? I believe it was successful but I still see 3 commits.

**Comment by nc-hurleyk** [comment]
- There should only be one commit
- Your commit should be one commit ahead of the `master`.  (In the terminal you can see this by typing `git log` - your single commit would appear after the last commit in master).

**Comment by nc-hurleyk** [comment]
👏

**Pull request merged**